substrate/primitives/runtime/src/lib.rs

```diff
@@ -1034,6 +1031,19 @@ impl OpaqueValue {
 	}
 }
 
+// TODO: Remove in future versions and clean up `parse_str_literal` in `sp-version-proc-macro`
+/// Deprecated `Cow::Borrowed()` wrapper.
+#[macro_export]
+#[deprecated = "Use Cow::Borrowed() instead of create_runtime_str!()"]
+macro_rules! create_runtime_str {
+	( $y:expr ) => {{
+		$crate::Cow::Borrowed($y)
+	}};
+}
+// Re-export for ^ macro, should be removed once macro is gone
+#[doc(hidden)]
+pub use alloc::borrow::Cow;
```

Comment:
This doesn't work?
```suggestion
		$crate::alloc::borrow::Cow::Borrowed($y)
	}};
}
```

Reply:
Even better to wrap it in a `mod __private` module, so others dont assume that it is part of the public API.

Reply:
Ah yes, you can also have a `__private_create_runtime_str` module if you want to keep stuff together.

It is good so user don't start using it in when it is recommended by rustc.

Anyway it is fine.